templates/email/bromley/_council_reference_alert_update.html

```diff
@@ -0,0 +1,3 @@
+[% IF title == 'Garden Subscription - New' %]
```

Comment:
It would seem appropriate to include this in any waste email, not just new garden subscriptions? `report.cobrand_data` will be `waste` for any waste report.